pyproject.toml

```diff
@@ -37,10 +37,10 @@ aea = 'aea.cli:cli'
 [tool.poetry.dependencies]
 python = ">=3.8,<3.11"
 base58 = ">=1.0.3,<3.0.0"
-jsonschema = "^4.0.0"
+jsonschema = ">=3.2.0,<5"
```

Comment:
Why from 3.2.0 to 5?

Reply:
it works ok and 3.2 required by external team